generators/app/templates/sequelize_transaction.ejs

```diff
@@ -0,0 +1,34 @@
+const { sequelize } = require('../models');
+const { modelName } = require('../models');
```

Comment:
This should be something like:
```suggestion
const { <% modelName %> } = require('../models');
```
Like this user can enter model name when select this option